### PR TITLE
rendering-elements translations

### PR DESCRIPTION
**Pull request by sonipreshita**
Translated **Rendering-Elements** page of the Docs section to Gujarati.

### CODE REVIEW

File: content/docs/rendering-elements.md [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
 [](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
+તે પૃષ્ઠ પર "Hello, world" દર્શાવે છે.
 
-## Updating the Rendered Element {#updating-the-rendered-element}
+## Rendered Element ને અપડેટ કરી રહ્યું છે {#updating-the-rendered-element}
 
-React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object). Once you create an element, you can't change its children or attributes. An element is like a single frame in a movie: it represents the UI at a certain point in time.
+React elements [અવ્યવસ્થિત](https://en.wikipedia.org/wiki/Immutable_object) છે. એકવાર તમે element બનાવી લો, પછી તમે તેના children અથવા attributes ને બદલી શકતા નથી. એક element મૂવીમાં એક ફ્રેમ જેવું છે: તે ચોક્કસ સમયે UI રજૂ કરે છે.
```
Comment (RupenAnjaria) [inline content/docs/rendering-elements.md]: અવ્યવસ્થિત is not a correct word, it should be અપરિવર્તીત

**Comment by sonipreshita** [comment]
Thank you @RupenAnjaria

**Comment by 3sanket3** [comment]
@sonipreshita can you please use `yarn` to install the dependencies? 

It seems you used `npm`, due to which it added `package-lock.json`. We already have equivalent `yarn.lock` , so we should not commit the `package-lock.json`.

If you haven't set up `yarn` anytime, https://yarnpkg.com/en/docs/install should help. Let me know if any issue.

For this PR, please delete the `package-lock.json` file and push the changes.

**Comment by ron4ex** [comment]
Or if you don't want to use `yarn`, just don't commit the file in the pull request changes. Commit only the ones that are relevant.

**Comment by sonipreshita** [comment]
@ron4ex 
I have committed changes and removed a package-lock.json file from the PR. Check this and let me know.

**Comment by tesseralis** [comment]
@RupenAnjaria @3sanket3 is this ready for another review?

**Comment by 3sanket3** [comment]
@tesseralis Yes it is and I am reviewing the same. 

We were little confused as we decided initially to translate 'Hello World' in native Gujarati but then we found at many places the documentation directly pulls code from the `.js` files where the code is obviously written in English. Also, referred other translation repositories and finally thinking to keep 'Hello World' as it is. We will also update the same thing in other translated pages as well.

**Pull request merged**